crates/criticaltrust/src/signatures/keychain.rs

```diff
@@ -32,6 +36,30 @@ impl Keychain {
         Ok(keychain)
     }
 
+    pub fn keys(&self) -> &HashMap<KeyId, PublicKey> {
+        &self.keys
+    }
+
+    pub fn revocation_info(&self) -> Option<RevocationInfo> {
+        self.revocation_info.clone()
+    }
+
+    /// Load the following into [`Keychain`] provided the [`KeysManifest`].
```

Comment:
This is a misleading documentation comment. It adds the keys from the manifest and **replaces** the `RevocationInfo`. It may be worth considering making it an error to call this when `self.revocation_info.is_some()`.

Reply:
Thank you! I have update the doc comment and actually added the error on is_some.